README.md

```diff
@@ -32,6 +32,8 @@ Custom Resource Definitions (CRDs) and Kubernetes-native tooling.
   Protect your services using authentication methods of your choice.
 - **Declarative configuration for Kong**
   Configure all of Kong using CRDs in Kubernetes and manage Kong declaratively.
+- **Gateway Discovery**
+  Dynamically provision and scale your Kong Gateway deployments for KIC to discover.
```

Comment:
```suggestion
  KIC monitors your Kong Gateway deployment and pushes configuration to all replicas.
```

Original wording suggests that KIC is actually scaling the gateway deployment rather than KIC being able to manage dynamic scale deployments (where scale/replica count is still managed externally).